lms/templates/certificates/accomplishment-base.html

```diff
@@ -27,11 +28,17 @@
 
     <%static:css group='style-certificates'/>
 
-    % if site_configuration:
-      <link rel="stylesheet" type="text/css" href="${site_configuration.get_css_url()}" />
+    % if style_overrides_file:
```

Comment:
I have no idea how's that solving the issue, since this is exactly the implementation of `SiteConfiguration.get_value()`:

```python
    def get_css_url(self):
        if settings.USE_S3_FOR_CUSTOMER_THEMES:
            kwargs = {
                'location': "customer_themes",
            }
            storage = get_storage_class()(**kwargs)
            return storage.url(self.get_value('css_overrides_file'))
        else:
            return static("customer_themes/{}".format(self.get_value('css_overrides_file')))
```

What do you think?

Reply:
@grozdanowski Ping! Please check @OmarIthawi Question here.

Reply:
@OmarIthawi to be completely honest, I don't really know. I basically copied the way the styles are included from a template in LMS (`head-extra.html`) guided by the logic "if it works there then that's the way it should work here". :/

Reply:
So, it's working on production now @grozdanowski right? I have a good reason to think that this was a non-solution for the devstack craziness we've been having.

I hope it's like that and we can ignore it.

Let me know if this is still something that's a must have for you, I can double check and debug further.

Reply:
This was definitely an issue with devstack before, but after pulling newest stuff is no longer there. Removing this PR.

Reply:
Amazing! Thanks @grozdanowski!